keras-to-hls/keras-config.yml

```diff
@@ -1,10 +1,10 @@
-KerasJson: example-keras-model-files/KERAS_3layer.json
-KerasH5:   example-keras-model-files/KERAS_3layer_70pruned_retrained_weights.h5
+KerasJson: example-keras-model-files/KERAS_3layer_ternary_small.json
```

Comment:
Do we really want to change the default example (intended for new users to get into hls4ml) to be ternary dense model?

Also, adding this to the automated tests would be nice.

Reply:
I agree. I changed the default example to the usual 3-layer model. And also added bnn/tnn to the list of models for automated test.